pkg/asset/installconfig/aws/availabilityzones.go

```diff
@@ -28,9 +29,18 @@ func availabilityZones(ctx context.Context, session *session.Session, region str
 		return nil, errors.Wrap(err, "fetching availability zones")
 	}
 
+	// This regex finds local zones and allows us to filter them out.
+	// This is a temporary fix until we update our aws-sdk-go package to a
+	// version that adds the ZoneType field to the AvailabilityZone struct
+	regex, err := regexp.Compile("^\\w+-\\w+-\\w+-\\w+-\\w+$")
```

Comment:
nit: because I'm not great at parsing regex, I find it helpful to have examples. 

```
us-east-1-bos-1a
--

```
us-east-1a
```